science/bali-phy/Portfile

```diff
@@ -1,110 +1,41 @@
 PortSystem 1.0
 
 name                bali-phy
-version             2.1.1
-revision            5
-description         Bayesian Alignment and Phylogeny estimation
-long_description    BAli-Phy can estimate phylogenetic trees from sequence data when the alignment \
-                    is uncertain. Instead of conditioning on a single alignment estimate, BAli-Phy \
-                    accounts for alignment uncertainty by integrating over all alignments. \
-                    BAli-Phy does not rely on a guide tree because the alignment and the tree are \
-                    co-estimated. Therefore it can construct phylogeny estimates of widely \
-                    divergent sequences without bias toward a guide tree.
+version             3.0-beta6
+revision            6
+description         Estimation of phylogeny and multiple alignment via MCMC
+long_description    Estimation of phylogeny and multiple alignment via MCMC \
+                    BAli-Phy that estimates multiple sequence alignments and \
+		    evolutionary trees from DNA, amino acid, or codon sequences.\
+		    BAli-Phy uses MCMC and Bayesian  methods to estimate \
+		    evolutionary trees, positive selection, and branch lengths \
+		    while averaging over alternative alignments. \
+		     \
+		    BAli-Phy can also estimate phylogenies from a fixed \
+		    alignment (like MrBayes and BEAST) using substitution \
+		    models like GTR+gamma.  BAli-Phy automatically estimates \
+		    relative rates for each gene.
 categories          science
 platforms           darwin
+license		    GPL-2
 maintainers         nomaintainer
-homepage            http://www.biomath.ucla.edu/msuchard/bali-phy/
+homepage            http://www.bali-phy.org/
 
-master_sites        ${homepage}
-checksums           sha1 e72073a1c5b05c797668e476bfd8517594f074e6
+# By using a SHA, this should be completely reproduceable.
+fetch.type	    git
+git.url		    https://github.com/bredelings/BAli-Phy.git
+git.branch	    7c46f800d533947b34142fdcf9597f14335b8c37
 
-patchfiles          configure.patch scripts.patch src-tools-draw-tree.patch
+depends_build       port:pkgconfig port:meson port:ninja
```

Comment:
Can you please try
```
PortGroup meson 1.0
```
?

It's new, so not really well tested and there are a number of known issues with meson anyway.